src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -207,12 +259,26 @@ private void ProcessDiscoveryFrame(Frame frame)
 				{
 					if (this.Log().IsEnabled(LogLevel.Debug))
 					{
-						this.Log().LogDebug($"Failed to create instace of server processor in  {asm} : {exc}");
+						this.Log().LogDebug($"Failed to create instance of server processor in  {asm} : {exc}");
```

Comment:
It seems weird that we're logging debug messages here, when previously we were logging error message when the assembly couldn't be loaded

Reply:
I'm not sure on the rules/guidance on when each should be used. This is likely only to be necessary/useful during development but have standardized with elsewhere.